CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This CHANGELOG follows the format listed at [Our CHANGELOG Guidelines](https://g
 Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Changed
```

Comment:
Because this crosses major version bumps and we use semantic versioning this constitutes a `### Breaking Change`.

It should read something like this:
```
### Breaking Changes
- bumped dependency of `sensu-plugin` to `>= 2.5 && < 3.0` you can read about it  [here](https://github.com/sensu-plugins/sensu-plugin/blob/master/CHANGELOG.md#v145---2017-03-07) (@CoRfr)
```